views/css/adyen.css

```diff
@@ -92,3 +92,9 @@ div.adyen-payment-method-label {
 div.adyen-additional-data-line {
     padding-left: 15px;
 }
+
+@supports (-webkit-appearance:-apple-pay-button) {
+    .adyen-checkout__applepay__button._1dIj5OBNHw-_NvtfqKwJuZ {
```

Comment:
do not use this id `_1dIj5OBNHw-_NvtfqKwJuZ` it might change in the next bundle release
Also please add a comment above this style why it was needed

Reply:
CSS selector updated